tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,136 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Basic aistack test
+
+# Summary: This test performs the following actions
+#  - Create a VM in EC2 using SLE-Micro-BYOS later than 6.0 version
+#  - Install the required dependencies to install the aistack helm chart
+#  - Test access to OpenWebUI and run integration tests with Ollama and MilvusDB
+# Maintainer: Yogalakshmi Arunachalam <yarunachalam@suse.com>
+#
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use publiccloud::utils qw(is_byos registercloudguest register_openstack);
+use publiccloud::ssh_interactive 'select_host_console';
+use transactional;
+use containers::k8s;
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use transactional qw(process_reboot trup_install trup_shell);
+use File::Basename;
+use version_utils;
+use Data::Dumper;
+sub install_dependency_package {
+    my ($instance, $rke2_url, $kubectl_url, $helm_url) = @_;
+    trup_call("pkg install curl git docker");
+    process_reboot(trigger => 1);
+    systemctl("enable docker");
+    systemctl("start docker");
+    systemctl("status docker");
+    script_run("curl -sSL $rke2_url -o ./install_rke2.sh && chmod 775 ./install_rke2.sh");
+    script_run("sh ./install_rke2.sh");
+    script_run("echo 'export PATH=\$PATH:/opt/rke2/bin' >> ~/.bashrc");
+    systemctl("enable rke2-server.service");
+    systemctl("start rke2-server.service");
+    systemctl("status rke2-server.service");
+    script_run("rke2 --version");
+    script_run("curl -sSL $helm_url -o ./install_helm.sh && chmod 775 ./install_helm.sh");
+    script_run("sh ./install_helm.sh");
+    script_run("helm version");
+    script_run("curl -sSL $kubectl_url -o ./kubectl && chmod +x ./kubectl");
+    script_run("sudo mv ./kubectl /usr/local/bin/");
+    script_run("kubectl version --client");
+}
+
+sub install_aistack_chart {
+    my ($instance, $cert_repo, $ing_repo) = @_;
+    record_info('Install AISTACK helm charts');
+    my $APPLICATION_COLLECTION_USER_PASSWORD_SECRET = "mysecretpassword";
+    my $docker_user_name = 'yarunachalam@suse.com';
+    my $user = $testapi::username;
+    my $email = "you\@example.com";
+    my $helm_values = get_var('HELM_CONFIG', 'https://github.com/yarunachalam/os-autoinst-distri-opensuse/tree/aistack_basic/data/aistack_values.yaml');
```

Comment:
Since your helmchart is already commited, you can do:
```suggestion
    my $helm_values = get_var('HELM_CONFIG', autoinst_url("/data/aistack_values.yaml"));
```

Reply:
converted this as a parameter